dockerfiles/benchmarks/push_bench_result.py

```diff
@@ -16,75 +17,110 @@
-                        type=str,
-                        required=True,
-                        help='Benchmark name')
-    parser.add_argument('--result','-r',
-                        type=str,
-                        required=True,
-                        help='Result of benchmark')
-    parser.add_argument('--labels','-l',
-                        type=str,
-                        help='Dictionary with additional labels')
-    parser.add_argument('--unit','-u',
-                        type=str,
-                        default='ns',
-                        help='Metric unit (ns,s,bytes,info...)')
-    parser.add_argument('--prometheus-server','-s',
-                        type=str,
-                        required=True,
-                        help='Prometheus server address http://prom.com')
+    parser = argparse.ArgumentParser(
+        description="Push data to Victoria Metrics."
+    )
+    parser.add_argument(
+        "--type",
+        "-t",
+        default="common",
+        choices=["common", "specific", "test"],
+        type=str,
+        help="Type of benchmark: common|specific|test",
+    )
+    parser.add_argument(
+        "--project",
+        "-p",
+        type=str,
+        required=True,
+        help="Benchmark project (usually name of the repo)",
+    )
+    parser.add_argument(
+        "--name", "-n", type=str, required=True, help="Benchmark name"
+    )
+    parser.add_argument(
+        "--result", "-r", type=str, required=True, help="Result of benchmark"
+    )
+    parser.add_argument(
+        "--labels", "-l", type=str, help="Dictionary with additional labels"
+    )
+    parser.add_argument(
+        "--unit",
+        "-u",
+        type=str,
+        default="ns",
+        help="Metric unit (ns,s,bytes,info...)",
+    )
+    parser.add_argument(
+        "--prometheus-server",
+        "-s",
+        type=str,
+        required=True,
+        help="Prometheus server address http://prom.com",
+    )
     args = parser.parse_args()
     return args
 
+
 def create_metric(args):
     """
     Creates metric in prometheus format from script arguments
     Common metric doesn't have labels, test metric can have labels
     """
     if args.labels:
-        if args.type == 'common':
+        if args.type == "common":
             print("Common metric shouldn't have additional labels")
             sys.exit(1)
-        metric = f"parity_benchmark_{args.type}_result_{args.unit}{{project=\"{args.project}\",benchmark=\"{args.name}\",{args.labels}}} {args.result}"
+        metric_name = f'parity_benchmark_{args.type}_result_{args.unit}{{project="{args.project}",benchmark="{args.name}",{args.labels}}}'
     else:
-        metric = f"parity_benchmark_{args.type}_result_{args.unit}{{project=\"{args.project}\",benchmark=\"{args.name}\"}} {args.result}"
-    return metric
+        metric_name = f'parity_benchmark_{args.type}_result_{args.unit}{{project="{args.project}",benchmark="{args.name}"}}'
```

Comment:
Will variables work in single quotes?

Reply:
Yes they work. I have examples of running all the scripts in the "details" section in the description of the MR and in the issue. Please check them.